### PR TITLE
Enable VideoPress block on Android

### PR DESCRIPTION
**Pull request by fluiddot**
**Related PRs:**
* https://github.com/wordpress-mobile/WordPress-Android/pull/18440

This PR enables the VideoPress block on Android.

**To test:**
1. Open the Android app and connect it with the local Metro server.
2. Open/create a post.
3. Tap ➕ button to open the block inserter.
4. Observe that the VideoPress block is displayed.
5. Tap on the VideoPress block to add it.
6. Add a video.
7. Observe that the video is displayed and can be played.

PR submission checklist:

- [ ] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/trunk/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/trunk/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

File: src/jetpack-editor-setup.js
Comment (SiobhyB) [inline src/jetpack-editor-setup.js:1]: Noting that the CI's currently failing due to the now-unused import in this file:

`import { Platform } from '@wordpress/element';`

**Review by SiobhyB — COMMENTED** [review]
@fluiddot, looking good! Happy to approve after the unused import is removed. I think the [supported-blocks.json](https://github.com/wordpress-mobile/gutenberg-mobile/blob/bf65922627fbc390151344a262afbf33bdb4f6a4/src/block-support/supported-blocks.json) file can also be updated to reflect that the block's available on both platforms.

File: src/jetpack-editor-setup.js
Comment (fluiddot) [reply src/jetpack-editor-setup.js:1]: Ah, good catch. I've fixed it in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5780/commits/deb8518405b2595a804918f940ec6f4773e7a92f.

**Comment by fluiddot** [comment]
> I think the [supported-blocks.json](https://github.com/wordpress-mobile/gutenberg-mobile/blob/bf65922627fbc390151344a262afbf33bdb4f6a4/src/block-support/supported-blocks.json) file can also be updated to reflect that the block's available on both platforms.

Good idea. I've updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5780/commits/f1904079da950deb35c101da222e1ba13e21bec2.

**Review by SiobhyB — APPROVED** [review]
LGTM! Thanks @fluiddot 🙌

**Pull request merged**